src/jit/emit.h

```diff
@@ -255,8 +255,6 @@ struct insGroup
     UNATIVE_OFFSET igNum;     // for ordering (and display) purposes
     UNATIVE_OFFSET igOffs;    // offset of this group within method
     unsigned int   igFuncIdx; // Which function/funclet does this belong to? (Index into Compiler::compFuncInfos array.)
-    unsigned short igFlags;   // see IGF_xxx below
```

Comment:
Why did you move these two (igFlags/igSize)? They should already be appropriately aligned/packed on all architectures.

Reply:
Note that `emitterStaticStats()` is one way to help figure out if the packing is tight. (Change
```
#define EMITTER_STATS 0       // Collect stats on the emitter.                       
```
to define to `1` and rebuild (in release), then run to see its output.

Reply:
The change affects only `defined(DEBUG) || defined(LATE_DISASM)`.

Note: in my opinion `/d1reportSingleClassLayoutinsGroup` is more useful, you can rebuild only clrjit project from VS and see the result much faster.

Reply:
I don't care about DEBUG. And `LATE_DISASM` is also not enabled by default. I didn't know about the switch you give; can you show sample output?

Reply:
There are two VS switches `/d1reportSingleClassLayout<name>` and `/d1reportAllClassLayout`.
Add them to CoreCLr.sln->clrjit->properties->C/C++ -> Command Line -> Additional Options:
![image](https://user-images.githubusercontent.com/22642771/66871723-1e3db880-ef59-11e9-8dad-7559b7469ecb.png)

and see the layout of the chosen struct when you build clrjit:
![image](https://user-images.githubusercontent.com/22642771/66871766-3f060e00-ef59-11e9-987d-36eb946cd0ef.png)